protobuf/build.gradle

```diff
@@ -10,19 +10,20 @@ plugins {
 description = 'gRPC: Protobuf'
 
 dependencies {
-    compile project(':grpc-api'),
+    api project(':grpc-api'),
+            libraries.jsr305,
             libraries.protobuf
-    guavaDependency 'compile'
+    guavaDependency 'implementation'
 
-    compile (libraries.google_api_protos) {
+    api (libraries.google_api_protos) {
         // 'com.google.api:api-common' transitively depends on auto-value, which breaks our
         // annotations.
         exclude group: 'com.google.api', module: 'api-common'
-        // Prefer our more up-to-date protobuf over 3.2.0
```

Comment:
The comment was this way so it would be obvious if it was out-of-date. It is very easy to upgrade a dependency, like google_api_protos, and fail to update to a newer version of the excluded dependency. So this comment was showing that we last verified when google_api_protos depended on protobuf 3.2.0.

I don't think we want to _blindly_ prefer our version of protobuf.

Reply:
Neither our version or that google_api_protos depends on is 3.2.0 now. What about 
```
// Prefer our more up-to-date version from from libraries.protobuf
```
?

Reply:
I'd suggest leaving it as-is for this PR, since it is an unrelated change.

But if you'd like to fix it, then change the version to "3.7.1", which is the current version brought in (and since we are using 3.11.0, everything is still fine with the exclusion).

Reply:
Leave it as-is now.